resources/manifest.json

```diff
@@ -7,9 +7,9 @@
   "options_page": "options.html",
 
   "icons": {
-    "16": "icon_off.png",
-    "48": "icon_off.png",
-    "128": "icon_off.png"
+    "16": "img/icon/icon_off.png",
+    "48": "img/icon/icon_off.png",
+    "128": "img/icon/icon_off.png"
```

Comment:
Yes, I'm going to be picky again :P The URL now has the word `icon` twice now, so I'd like `img/icon/off.png` better :)